docs/source/howto/data.rst

```diff
@@ -514,7 +514,149 @@ To learn about more advanced queries, please see :ref:`the corresponding topics
 Organizing data
 ===============
 
-`#3997`_
+
+.. _how-to:data:organize:grouppath:
+
+Organise groups in hierarchies
+------------------------------
+
+.. meta::
+   :description: The reStructuredText plaintext markup language
+   :keywords: grouppath
+
+Groups in AiiDA are inherently "flat", in that groups may only contain nodes and not other groups.
+However, the `GroupPath` utility allows one to construct *virtual* group hierarchies based on delimited group labels.
+
+`GroupPath` is designed to work in much the same way as python's `pathlib.Path`, whereby paths are denoted by forward slash characters '/' in group labels.
```

Comment:
```suggestion
`GroupPath` is designed to work in much the same way as Python's `pathlib.Path`, whereby paths are denoted by forward slash characters '/' in group labels.
```

Reply:
Can you link both `GroupPath` and `pahtlib.Path` to the respective API reference docs?